paying-for-college/disclosures/admin.py

```diff
@@ -1,21 +1,29 @@
 #!/usr/bin/env python
 from __future__ import absolute_import
 from django.contrib import admin
-from .models import School, Alias, BAHRate, Feedback, Worksheet
+from .models import School, Program, Alias, Nickname, Contact
+from .models import BAHRate, Feedback, Worksheet
 
 
 class SchoolAdmin(admin.ModelAdmin):
-    list_display = ('primary_alias', 'school_id', 'city', 'state')
-    search_fields = ['primary_alias']
+    list_display = ('primary_alias', 'school_id', 'city', 'state', 'KBYOSS')
```

Comment:
This is super, outrageously nit-picky, but I don't think I'd know (in fact, I don't), what `KBYOSS` means. Perhaps a comment that explains it?